user-guide/Reference/Skyline_DataMiner_Folder/More_information_on_certain_files_and_folders/ConnectionSettings_txt.md

```diff
@@ -38,9 +38,11 @@ This file contains default connection settings to be used by DataMiner client ap
 
   Possible values:
 
-  - *GRPCConnection*: Supported from DataMiner 10.3.0/10.3.2 onwards. DataMiner will communicate using HTTPS via the API Gateway, using gRPC GZIP compression. By default, this requires the use of the standard HTTPS port 443. <!-- RN 34983 -->
+  - *GRPCConnection*: Supported from DataMiner 10.3.0/10.3.2 onwards. DataMiner will communicate using HTTPS via the API Gateway, using gRPC. By default, this requires the use of the standard HTTPS port 443. <!-- RN 34983 -->
 
-  - *RemotingConnection*: .NET Remoting.
+  - *RemotingConnection*: .NET Remoting - Obsolete starting from 10.4.0/10.3.6 onwards. However, this will remain the default until .NET remoting is disabled by default <!-- RN 36196 - Once .NET Remoting gets disabled by default, an upgrade action will update these values to GRPCConnection while LegacyRemotingConnection types will be left as-is -->
```

Comment:
```suggestion
  - *RemotingConnection*: .NET Remoting. Obsolete starting from 10.4.0/10.3.6. While DataMiner will currently not yet automatically switch to *GRPCConnection*, this is planned to be implemented with a DataMiner upgrade soon. If you do not want to use *GRPCConnection*, use *LegacyRemotingConnection* to avoid getting automatically switched. However, note that we strongly recommend using *GRPCConnection*.<!-- RN 36196 -->
```